### PR TITLE
Milestone 2: User interface

### PR DESCRIPTION
**Pull request by evansinho**
####  this milestone we created:
- An interface that interacts with the user of the application.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
# Hi team! :clap: 
### Good job in Milestone 2 :eyes: 
### Just few changes before it can be approved :100: 

 [REQUIRED] game informs player if selected move is `invalid`

##### Implementation details

* [ ] A loop is needed in this game (recommended is `while` but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png))
* [ ] Inside loop, you should all actions repeated for each move
* [ ] A loop needs to be terminated at some point (after a winning move or draw to be more specific). It can be done with variable:

* Using out-of-nowhere numbers to terminate a loop, for example `5.times do ... end` is a signal that something is wrong. Advise usage of terminating variable as above.

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding * <g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png">computer</g-emoji>

**Comment by evansinho** [comment]
Hello, @Anna-Myzukina  thanks for the review.
but this changes you requested for are already there on the codebase. Kindly check properly again
Thanks.

File: bin/main.rb
Comment (kobiyoyo) [inline bin/main.rb]: The loop needs to be terminated with a variable,not a number

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team     
#### Great Job so far, but there are a few more things required. 
### General
- [x]  [REQUIRED] game displays board after player move
- [x] Lets include `lib` directory .
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by evansinho** [comment]
The board is displayed after a player move.

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @evansinho and @rajkumardas2701 
### You have done really well however, you have conflict you need to solve before another review.
conflict
- Solve the conflict and resubmit for another code review. Good luck
- Happy coding!

**Comment by rajkumardas2701** [comment]
Hi Raymond/Team,

We are awaiting approval/feedback before we can merge our user_interface branch to Development.
We will also take care of conflict then :)

Thanks and Regards,
Rajkumar Das

**Comment by uryelah** [comment]
Hello **Evanson Igiri & Rajkumar Das Das**, I'm Sarah, your last reviewer for this PR 👩‍🚀  

Good job fixing that conflict.

---------

## **Your capstone is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---------

**Keep being awesome devs and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**
